### ISSUE TITLE
Freeze the Discriminator: a Simple Baseline for Fine-Tuning GANs

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/paper-notes#13]
- 적은 데이터 도메인에 GAN fine-tuning 할 때 D의 lower-layer를 freeze하면 overfit 우려 감소

## Rel works
GAN transfer는 크게 네 가지 카테고리로 분류 가능
- Fine-tuning: G, D 모두 pre-trained model에서 가져오거나, G만 가져오기; overfit 잘됨
- Scale/shift: batch norm stats만 fine-tune하기; 그런데 src, tgt 도메인이 매우 다르면 잘 안됨
- Generative latent optimization: G와 latent code 동시에 opt; stability는 좋아지는데 이미지가 blurry 해짐 (gan loss 안써서)
- MineGAN: G fix하고 latent code만 mining함
